src/pages/blogPost/BlogPost.jsx

```diff
@@ -1,23 +1,61 @@
 import './BlogPost.css'
-import {useParams} from "react-router-dom";
-import {useState} from "react";
+import {Link, useParams} from "react-router-dom";
+import ConvertDate from "../../assets/helpers/ConvertDate.jsx";
+import {useEffect, useState} from "react";
+import axios from "axios";
+
+function BlogPost() {
 
-function BlogPost () {
-    /*const [postInfo, setPostInfo]   = useState({})*/
     const {postID} = useParams();
     console.log(postID)
 
-/*function fetchPost(){
-    setPostInfo(data)
-}*/
+    const [posts, setPosts] = useState([])
+    const [error, setError] = useState(null)
+
```

Comment:
hetzelfde als het vorige punt json importeren en dan navigeren is schoner dan axios